apps/web/src/pages/integrations/useProviders.ts

```diff
@@ -72,17 +76,17 @@ function initializeProvidersByIntegration(integrations: IntegrationEntity[]): II
     fcmFallback(integrationItem, clonedCredentials);
 
     return {
-      providerId: providerItem.id,
+      providerId: integrationItem.providerId || providerItem?.id,
```

Comment:
I do not understand why in this return some of the fallbacks are prioritised by `integrationItem` and others `providerItem`. Could you share the reasoning of it? I am confused, sorry. 😞

Reply:
Because Novus free provider is not a part of the novu providers list and then we need to use it for the integration item we get from the backend.